nexus/src/external_api/http_entrypoints.rs

```diff
@@ -2196,6 +2365,9 @@ async fn hardware_sleds_get_sled(
 
 /**
  * Refresh update metadata
+ *
+ * Refreshes the update metadata. If there is a new software update, then the server
+ * will be aware of it and can optionally perform the update.
```

Comment:
@iliana likely you know more of the details we can provide here as to the behavior of this endpoint, or should we hide this endpoint from external users.

Reply:
Might make sense to hide it for now, but long-term I think operators should have a "Check for new updates" button in the console that would trigger a metadata refresh and possibly rebuilding the update plan. (See also #761)

Reply:
ah okay dope